sakai-evaluation-api/src/java/org/sakaiproject/evaluation/logic/model/HierarchyNodeRule.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2005 Sakai Foundation Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ * http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package org.sakaiproject.evaluation.logic.model;
+
+import java.io.Serializable;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.sakaiproject.evaluation.model.EvalHierarchyRule;
+
+/**
+ * This represents a single hierarchy rule in the system.
+ */
+public class HierarchyNodeRule implements Serializable
+{
+    private static final long serialVersionUID = 1L;
+    private static final Log LOG = LogFactory.getLog( HierarchyNodeRule.class );
+
+    private Long id;        // unique identifier
+    private Long nodeID;    // the node this rule belongs to
+    private String rule;    // the actual rule text (with qualifiers applied)
+    private String option;  // the 'option' for the rule (site or section)
```

Comment:
Maybe this would be better as an enum with defined valid values rather than a freeform string.